libcontainer/container_linux.go

```diff
@@ -442,12 +443,20 @@ func (c *linuxContainer) newParentProcess(p *Process) (parentProcess, error) {
 	if err != nil {
 		return nil, newSystemErrorWithCause(err, "creating new init pipe")
 	}
-	cmd, err := c.commandTemplate(p, childPipe)
+	messagePipe := readWritePair{parentPipe, childPipe}
```

Comment:
Should really be called `messageSockPair` or similar.

Reply:
Done